exporter/info.go

```diff
@@ -100,11 +100,12 @@ func (e *Exporter) extractInfoMetrics(ch chan<- prometheus.Metric, info string,
 			continue
 
 		case "Keyspace":
-			if keysTotal, keysEx, avgTTL, ok := parseDBKeyspaceString(fieldKey, fieldValue); ok {
+			if keysTotal, keysEx, avgTTL, keysCached, ok := parseDBKeyspaceString(fieldKey, fieldValue); ok {
 				dbName := fieldKey
 
 				e.registerConstMetricGauge(ch, "db_keys", keysTotal, dbName)
 				e.registerConstMetricGauge(ch, "db_keys_expiring", keysEx, dbName)
+				e.registerConstMetricGauge(ch, "db_keys_cached", keysCached, dbName)
```

Comment:
Sorry, I missed this earlier.
This will start registering a new metric for all types of targets, even if it's not a KeyDB instance (and vanilla Redis doesn't have cached keys, right?).
Can you modify this to only emit the metric if it was found? 
Could be as simple as initializing `keysCachedTotal = -1` in line 224 and then add a check around line 108 to only record the metric if `keysCached > -1`

Reply:
Done, but the test coverage is decreasing.

Reply:
I think this is fine, thanks for updating the code!